console/src/main/scripts/plugins/metrics/html/alerts-center-triggers.html

```diff
@@ -0,0 +1,93 @@
+            <label><input type="search" ng-model="act.search" aria-controls="DataTables_Table_0"></label>
+          </div>
+          <div class="dataTables_info" id="DataTables_Table_0_info" role="status" aria-live="polite">Showing <b>1</b> to <b>{{(act.triggersList | filter:act.search).length}}</b> of <b>{{act.triggersList.length}}</b> Items</div>
+        </div>
+        <table class="datatable table table-bordered hk-table-alerts hk-table-detailed hk-table-select">
+          <thead>
+          <tr>
+            <th ng-click="act.selectAll()"><input type="checkbox"/></th>
+            <th ng-class="{'sorting_asc': act.sortField == 'name' && act.sortAsc, 'sorting_desc': act.sortField == 'name' && !act.sortAsc, 'sorting': act.sortField != 'name'}" ng-click="act.sortBy('name')">Name</th>
+            <th ng-class="{'sorting_asc': act.sortField == 'description' && act.sortAsc, 'sorting_desc': act.sortField == 'description' && !act.sortAsc, 'sorting': act.sortField != 'description'}" ng-click="act.sortBy('description')">Description</th>
+            <th ng-class="{'sorting_asc': act.sortField == 'context.resourceName' && act.sortAsc, 'sorting_desc': act.sortField == 'context.resourceName' && !act.sortAsc, 'sorting': act.sortField != 'context.resourceName'}" ng-click="act.sortBy('context.resourceName')">Resource</th>
+            <th ng-class="{'sorting_asc': act.sortField == 'severity' && act.sortAsc, 'sorting_desc': act.sortField == 'severity' && !act.sortAsc, 'sorting': act.sortField != 'severity'}" ng-click="act.sortBy('severity')" ng-click="act.sortBy('severity')">Severity</th>
+            <th ng-class="{'sorting_asc': act.sortField == 'enabled' && act.sortAsc, 'sorting_desc': act.sortField == 'enabled' && !act.sortAsc, 'sorting': act.sortField != 'enabled'}" ng-click="act.sortBy('enabled')" ng-click="act.sortBy('enabled')">Enabled</th>
+            <th ng-class="{'sorting_asc': act.sortField == 'notifications' && act.sortAsc, 'sorting_desc': act.sortField == 'notifications' && !act.sortAsc, 'sorting': act.sortField != 'notifications'}" ng-click="act.sortBy('notifications')" ng-click="act.sortBy('notifications')">Notifications</th>
+            <th></th>
+          </tr>
+          </thead>
+          <tbody>
+          <tr ng-repeat="trigger in act.triggersList | filter:act.search" ng-class="{'hk-selected': act.selected}"
+              ng-click="act.selectItem(trigger)">
+            <td><input type="checkbox" ng-checked="trigger.selected"/></td>
+            <td>{{trigger.name}}</td>
+            <td>{{trigger.description}}</td>
+            <td>{{trigger.context.resourceName|truncate: 30}}</td>
+            <td>{{trigger.severity|firstUpper}}</td>
+            <td>{{trigger.enabled ? "True" : "False"}}</td>
+            <td>{{trigger.actions.email.length}} <i class="fa fa-envelope"></i></td>
+            <td><a class="btn btn-link"><i class="fa fa-chevron-circle-right fa-lg"
```

Comment:
The PR looks good to me.
The only comment I have is that as this is going to be iterative, if for this cut we are not going to add the detail page I should hide/comment the icon to not present a link that has not defined the alert page.
So in that case, the MS6 would have the trigger list but that's all (and it's ok) we can complete it later.
But it's a minor comment, no blocker, I'm +1 to merge it as it is and refine this change later of this PR.